eyre/tests/test_location.rs

```diff
@@ -155,7 +157,7 @@ fn test_option_compat_wrap_err_with() {
     }));
 
     use eyre::ContextCompat;
-    let err = None::<()>.wrap_err_with(|| "oopsie").unwrap_err();
+    let err = None::<()>.with_context(|| "oopsie").unwrap_err();
```

Comment:
The encosing test `test_option_compat_wrap_err_with` might better be renamed to `test_option_compat_with_context`?